bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/utils/IdentifierUtils.java

```diff
@@ -40,4 +42,23 @@ public static String decodeIdentifier(String encodedIdentifier) {
         return Utils.decodeIdentifier(encodedIdentifier);
     }
 
+    /**
+     * Escape the special characters in an identifier with a preceding `\`.
+     *
+     * @param identifier encoded identifier string
+     * @return decoded identifier
+     */
+    public static String escapeSpecialCharacters(String identifier) {
+        return Utils.escapeSpecialCharacters(identifier);
+    }
+
+    /**
+     * Unescapes a ballerina string.
+     *
+     * @param text ballerina string to unescape
+     * @return unescaped ballerina string
```

Comment:
```suggestion
     * @return ballerina string with any escaped characters converted to their unescaped form
```
Just wanted to improve the understandability. May change as you wish by adding something like this to the description instead of here.